standards/out-of-hours.md

```diff
@@ -0,0 +1,60 @@
+---
+category: Operating services
+---
+# Supporting services out of hours
+
+Many of our services are used outside normal office hours (9am-6pm, Monday-Friday). To make sure people are able to keep using them with confidence, even when problems arise, it's sometimes necessary to have people on call out of hours to resolve these problems.
+
+Some services don't need out of hours support. This might be because their users don't rely on it for critical parts of their job, because the users themselves don't need it outside normal office hours, or because the service is robust enough to recover from a number of classes of failure itself.
+
+There are also cases where services may need out of hours support, but we're not currently able to provide it without risking the health and well-being of our staff, usually because we don't have enough staff who are able to support it. In these cases, we should either engage a wider group of staff to support a set of services, or seek the help of suppliers to provide sustainable support.
```

Comment:
I wonder if this statement is necessary for formal guidance?  Perhaps it is sufficient only to say that we don't currently have enough staff who are able to support it?

Reply:
Did highlight support disappear from GitHub, or was I imagining it... it's been a while!  By 'this statement' I meant: "without risking the health and well-being of our staff"